src/components/Methods.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import '../index.css';
+
+import img1 from '../images/Icon-1.png'
+import img2 from '../images/Icon-2.png'
+import img3 from '../images/Icon-3.png'
+import img4 from '../images/Icon-4.png'
+
+const Methods = ({ title, img1Title, img2Title, img3Title, img4Title }) => {
+  return (
+    <section className="methods-section">
```

Comment:
The Method section looks good, you just need a flexbox or grid to make it look more like the original design.